documentation/movies.workspace.adoc

```diff
@@ -702,6 +699,22 @@ MATCH path = (person)-[:ACTED_IN]->(m)<-[:DIRECTED]-(d)
 RETURN path;
 ----
 
+To aid query performance, you can add indexes to your data.
```

Comment:
Either move this to the beginning, but we should ensure that people execute this cypher-block, or keep it in the MERGE statement.